File: C/Easy/FizzBuzz/FizzBuzz.c
#include <stdio.h>
int main()
{
    long long divOne, divTwo, range;
   
    while (scanf("%lld%lld%lld", &divOne, &divTwo, &range) == 3)
    {
        for (int i = 1; i <= range; i++)
        {
            if (i%divOne == 0 && i%divTwo == 0) printf("FizzBuzz\n");
            else if (i%divOne == 0) printf("Fizz\n");
            else if (i%divTwo == 0) printf("Buzz\n");
            else printf("%d\n", i);         
        }      
    }
}

File: Java/Easy/FizzBuzz/FizzBuzz.java
import java.util.Scanner;

/**
 * FizzBuzz
 */
public class FizzBuzz {

    public static void main(String[] args) {
        try
        {
            long divOne, divTwo, range;
            Scanner sc = new Scanner(System.in);
            

            while(sc.hasNextLong())
            {
                divOne = sc.nextLong();
                divTwo = sc.nextLong();
                range = sc.nextLong();
                
                for (int i = 1; i <= range; i++) {
                    if (i%divOne == 0 && i%divTwo == 0) System.out.println("FizzBuzz");
                    else if(i%divOne == 0) System.out.println("Fizz");
                    else if(i%divTwo == 0) System.out.println("Buzz");
                    else System.out.println(i);
                }
                
            }
            sc.close();

        }
        catch(Exception e)
        {
            System.out.println("Something happend \n");
            e.printStackTrace();
        }
    }
    
}